Rakefile

```diff
@@ -86,3 +86,211 @@ EOM
   end
 end
 
+# ACCEPTANCE TEST RAKE TASKS + HELPERS
+
+namespace :acceptance do
+  require 'puppet_litmus/rake_tasks'
+  require_relative './spec/support/acceptance/litmus_helpers'
+
+  def cmdb_record_ids
+    inventory_hash = LitmusHelpers.inventory_hash_from_inventory_file
+    vars = inventory_hash['vars'] || {}
+    vars['cmdb_record_ids'] || {}
+  end
+
+  desc 'Provisions the VMs. This is currently just the master'
+  task :provision_vms do
+    if File.exist?('inventory.yaml')
+      # Check if a master VM's already been setup
+      begin
+        uri = on_master(&:uri)
+        puts("A master VM at '#{uri}' has already been set up")
+        next
+      rescue TargetNotFoundError
+        # Pass-thru, this means that we haven't set up the master VM
+      end
+    end
+
+    provision_list = ENV['PROVISION_LIST'] || 'acceptance'
+    Rake::Task['litmus:provision_list'].invoke(provision_list)
+  end
+
+  # TODO: This should be refactored to use the https://github.com/puppetlabs/puppetlabs-peadm
+  # module for PE setup
+  desc 'Sets up PE on the master'
+  task :setup_pe do
+    on_master { |master| master.bolt_run_script('spec/support/acceptance/install_pe.sh') }
+  end
+
+  desc 'Sets up the ServiceNow instance'
+  task :setup_servicenow_instance do
```

Comment:
This task could also take a `config` hash containing dev instance config and just set-up the `inventory.yaml` file.